recipes/mingw-w64/linux/conanfile.py

```diff
@@ -257,16 +260,17 @@ def build(self):
 
     def package(self):
         self.copy("COPYING", src=os.path.join(self.build_folder, "sources", "mingw-w64"), dst="licenses")
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "x86_64-w64-mingw32", "lib"), "*.la")
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "x86_64-w64-mingw32", "lib32"), "*.la")
-        tools.remove_files_by_mask(os.path.join(self.package_folder, "bin", "gcc", "x86_64-w64-mingw32", "10.3.0"), "*.la")
+        tools.remove_files_by_mask(self.package_folder, "*.la")
         tools.rmdir(os.path.join(self.package_folder, "share", "man"))
         tools.rmdir(os.path.join(self.package_folder, "share", "doc"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-        # Remove symlinks, we need to create them in the package_info step
+        # replace with relative symlinks so they'll resolve correctly on consumer's machine
         os.unlink(os.path.join(self.package_folder, 'mingw'))
         os.unlink(os.path.join(self.package_folder, self._target_tag, 'lib64'))
+        self.run("ln -s {} {}".format(os.path.join(os.curdir, self._target_tag),
+                                        os.path.join(self.package_folder, 'mingw')))
+        self.run("ln -s {} {}".format(os.path.join(os.curdir, 'lib'),
+                                        os.path.join(self.package_folder, self._target_tag, 'lib64')))
```

Comment:
why not using [`tools.fix_symlinks()`](https://docs.conan.io/en/latest/reference/tools.html#tools-fix-symlinks) ?

Reply:
just in case, calling `os.symlink` should be more portable than `ln -s` command line.

Reply:
`tools.fix_symlinks()` doesn't work:
```
ERROR: The file is a broken symlink, verify that you are packaging the needed destination files: '/home/conan/.conan/data/mingw-w64/8.0.2/_/_/package/4cfe33ba23f3b0106f09220780349c6834f3da6e/lib/libcc1.so.0'.You can skip this check adjusting the 'general.skip_broken_symlinks_check' at the conan.conf file.
```
the link is generated relative to the package folder and not the link itself:
```
ls -l /home/conan/.conan/data/mingw-w64/8.0.2/_/_/package/4cfe33ba23f3b0106f09220780349c6834f3da6e/lib/libcc1.so.0
lrwxrwxrwx 1 conan 1001 19 Feb  7 08:24 /home/conan/.conan/data/mingw-w64/8.0.2/_/_/package/4cfe33ba23f3b0106f09220780349c6834f3da6e/lib/libcc1.so.0 -> lib/libcc1.so.0.0.0
```